<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorWidget</class>
 <widget class="QWidget" name="EditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>435</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basics">
      <attribute name="title">
       <string>Basics</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0">
       <item row="0" column="2">
        <widget class="QRadioButton" name="applyTitle">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="artist"/>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="applyArtist">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelAlbum">
         <property name="text">
          <string>Album</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelGenre">
         <property name="text">
          <string>Genre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="album"/>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="applyAlbum">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelYear">
         <property name="text">
          <string>Year</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelNumber">
         <property name="text">
          <string>Number</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QRadioButton" name="applyDisc">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelArtist">
         <property name="text">
          <string>Artist</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="originalArtist"/>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSpinBox" name="year">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="applyYear">
           <property name="toolTip">
            <string>Apply value to all selected items</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelDisc">
           <property name="text">
            <string>Disc</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="disc"/>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QSpinBox" name="number">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelMaxNumber">
           <property name="text">
            <string>/</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="maxnumber">
           <property name="maximum">
            <number>999</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="autoNumbers">
           <property name="toolTip">
            <string>Automaticly number files in this directory</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="genre"/>
       </item>
       <item row="11" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="2">
        <widget class="QRadioButton" name="applyGenre">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="labelComment">
         <property name="text">
          <string>Comment</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="comment"/>
       </item>
       <item row="6" column="2">
        <widget class="QRadioButton" name="applyComment">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="title"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelUrl">
         <property name="text">
          <string>URL</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="url"/>
       </item>
       <item row="7" column="2">
        <widget class="QRadioButton" name="applyUrl">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelTitle">
         <property name="text">
          <string>Title</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="composer"/>
       </item>
       <item row="10" column="1">
        <widget class="QLineEdit" name="encoder"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="labelOriginalArtist">
         <property name="text">
          <string>Orig. artist</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="labelComposer">
         <property name="text">
          <string>Composer</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="labelEncoder">
         <property name="text">
          <string>Encoder</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QRadioButton" name="applyOriginalArtist">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QRadioButton" name="applyComposer">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QRadioButton" name="applyEncoder">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QRadioButton" name="applyMaxNumber">
         <property name="toolTip">
          <string>Apply value to all selected items</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pictures">
      <attribute name="title">
       <string>Pictures</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="images">
         <property name="acceptDrops">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragEnabled">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DropOnly</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>96</width>
           <height>96</height>
          </size>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideNone</enum>
         </property>
         <property name="verticalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerPixel</enum>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="gridSize">
          <size>
           <width>96</width>
           <height>98</height>
          </size>
         </property>
         <property name="viewMode">
          <enum>QListView::ListMode</enum>
         </property>
         <property name="uniformItemSizes">
          <bool>false</bool>
         </property>
         <property name="batchSize">
          <number>100</number>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0">
         <item row="1" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Description</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="imageDescription"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="imageType"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Type</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="addImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Add new picture</string>
           </property>
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeImage">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Remove picture</string>
           </property>
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportImage">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Save picture to disk</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::MinimumExpanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="applyImages">
           <property name="toolTip">
            <string>Apply pictures to all selected tracks</string>
           </property>
           <property name="text">
            <string>Apply to all</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="extra">
      <attribute name="title">
       <string>Extra</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Length</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_21">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>Bitrate</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_22">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_18">
           <property name="text">
            <string>Sample rate</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_20">
           <property name="text">
            <string>Frequency</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_24">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_19">
           <property name="text">
            <string>Filesize</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_25">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>title</tabstop>
  <tabstop>applyTitle</tabstop>
  <tabstop>artist</tabstop>
  <tabstop>applyArtist</tabstop>
  <tabstop>album</tabstop>
  <tabstop>applyAlbum</tabstop>
  <tabstop>year</tabstop>
  <tabstop>applyYear</tabstop>
  <tabstop>disc</tabstop>
  <tabstop>applyDisc</tabstop>
  <tabstop>number</tabstop>
  <tabstop>maxnumber</tabstop>
  <tabstop>genre</tabstop>
  <tabstop>applyGenre</tabstop>
  <tabstop>comment</tabstop>
  <tabstop>applyComment</tabstop>
  <tabstop>url</tabstop>
  <tabstop>applyUrl</tabstop>
  <tabstop>originalArtist</tabstop>
  <tabstop>applyOriginalArtist</tabstop>
  <tabstop>composer</tabstop>
  <tabstop>applyComposer</tabstop>
  <tabstop>encoder</tabstop>
  <tabstop>applyEncoder</tabstop>
  <tabstop>images</tabstop>
  <tabstop>imageType</tabstop>
  <tabstop>imageDescription</tabstop>
  <tabstop>applyImages</tabstop>
  <tabstop>addImage</tabstop>
  <tabstop>removeImage</tabstop>
  <tabstop>exportImage</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>imageType</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>EditorWidget</receiver>
   <slot>setImageType(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageDescription</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditorWidget</receiver>
   <slot>setImageDescription(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeImage</sender>
   <signal>clicked()</signal>
   <receiver>EditorWidget</receiver>
   <slot>removeCurrentImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addImage</sender>
   <signal>clicked()</signal>
   <receiver>EditorWidget</receiver>
   <slot>addImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportImage</sender>
   <signal>clicked()</signal>
   <receiver>EditorWidget</receiver>
   <slot>exportCurrentImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>366</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>autoNumbers</sender>
   <signal>clicked()</signal>
   <receiver>EditorWidget</receiver>
   <slot>automaticNumbering()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setImageDescription(QString)</slot>
  <slot>setImageType(QString)</slot>
  <slot>exportCurrentImage()</slot>
  <slot>addImage()</slot>
  <slot>removeCurrentImage()</slot>
  <slot>automaticNumbering()</slot>
 </slots>
</ui>
